java/spark-2.0.0/utils.scala

```diff
@@ -0,0 +1,19 @@
+package sparklyr
+
+import org.apache.spark.sql._
+
+object Utils200 {
```

Comment:
@kevinykuo could we rename this object to `Repartition`, `RepartitionUtils` or something like that? I think we should avoid versioned if possible.

From the code, it looks to me like we are compiling `Utils.scala` for Spark 1.X, could you consider using `scala_filter` under [spark_default_compilation_spec](https://github.com/rstudio/sparklyr/blob/8da6b058848ae9a3a47b95bd981bc509032f02c6/R/spark_compile.R#L236-L263) to avoid compiling any `2.x` files under `1.x`?